tests/yamlparser/fixtures/active-choices-param001.xml

```diff
@@ -12,42 +12,20 @@
       <parameterDefinitions>
         <org.biouno.unochoice.ChoiceParameter>
           <script>
-            <secureScript>
-              <script>return [
-    'param1',
-    'param2'
-]</script>
+            <secureFallbackScript>
+              <script/>
               <sandbox>true</sandbox>
-              <classpath>
-                <entry>
-                  <url>file:/jar-file-path</url>
-                </entry>
-                <entry>
-                  <url>file:/jar-file-path2</url>
-                </entry>
-              </classpath>
-            </secureScript>
+            </secureFallbackScript>
           </script>
           <script>
```

Comment:
In the old implementation <secureFallbackScript> and <secureScript> are children of just one <script>, not two separate <script> tags.